src/main/ui/app/route/Show.jsx

```diff
@@ -39,61 +39,128 @@ export default class Show extends Component {
+                               );
+
+            const versionmeaning = (
+                                <Tooltip id="tooltip"><strong>{addon.name} version</strong></Tooltip>
+                            );
+	    const versioncountmeaning = (
+                                <Tooltip id="tooltip"><strong>No. of builds of {addon.name}</strong></Tooltip>
+                            );
+            const requirementmeaning = (
+                                <Tooltip id="tooltip"><strong>This shows the minimum version of OpenMRS required to be able to run this module</strong></Tooltip>
+                            );
+	    let check="";
             let hosted = "";
+            let hostedtext="";
             if (addon.hostedUrl) {
                 if (addon.hostedUrl.includes("bintray.com")) {
-                    hosted = <p>Hosted on <a target="_blank" href={addon.hostedUrl}>Bintray</a></p>
+                    hostedtext="Hosted on";
+                    hosted = <a target="_blank" href={addon.hostedUrl}>Bintray</a>
                 }
                 else {
-                    hosted = <p>Hosted at <a target="_blank" href={addon.hostedUrl}>{addon.hostedUrl}</a></p>
+                    hostedtext="Hosted at";
+                    hosted =<a target="_blank" href={addon.hostedUrl}>{addon.hostedUrl}</a>
                 }
+	    check=(<tr>
+    <th>{hostedtext}</th>
+    <td><h5>{hosted}</h5></td>
+
+</tr>);
             }
-            let statusClass = addon.status === "ACTIVE" ? "label label-default" :
-                              addon.status === "DEPRECATED" ? "label label-danger" : "label label-warning";
+          
             return (
-                    <div>
-                        <h1>
-                            <i className={icon} aria-hidden="true"></i>
-                            {addon.name}
-                            <span className={statusClass}>{addon.status}</span>
-                        </h1>
-                        <h3>{addon.description}</h3>
-                        {hosted}
-                        <p>
-                            Maintained by:
-                            {addon.maintainers.map(m => {
+
+                    <div className="showpage-body">
+<Panel header={showtitle}>
+<h4 className="lead">{addon.description}</h4>
+<div className="col-md-6 col-sm-12 col-xs-12 left-margin">
+<Table striped condensed hover>
+<tbody>
+<tr>
+<th>Type :</th>
+<td><h5>{addon.type}</h5></td>
+</tr>
+{check}
+<tr>
+        <th>Maintained by: </th>
+        <td><h5>{addon.maintainers.map(m => {
                                 if (m.url) {
                                     return (
-                                            <a target="_blank" href={m.url}>{m.name}</a>
-                                    )
+                                    
+                                   <Button bsSize="small" href={m.url}>{m.name}</Button>                                   )
```

Comment:
I haven't looked at how this looks in the UI yet, but why would we display this as a button?